modules/cluster-logging-release-notes-5.3.z.adoc

```diff
@@ -0,0 +1,132 @@
+//Z-stream Release Notes by Version
+
+[id="cluster-logging-release-notes-5-3-3"]
+= OpenShift Logging 5.3.3
+This release includes link:https://access.redhat.com/errata/RHSA-2022:0227[RHSA-2022:0227 OpenShift Logging Bug Fix Release 5.3.3]
+
+[id="openshift-logging-5-3-3-bug-fixes"]
+== Bug fixes
+* Before this update, changes to the metrics dashboards had not yet been deployed because the cluster-logging-operator did not correctly compare existing and desired configmaps containing the dashboard.  This update fixes the logic by adding a the dashboard unique hash value to the object labels.(link:https://issues.redhat.com/browse/LOG-2066[LOG-2066])
```

Comment:
```suggestion
* Before this update, changes to the metrics dashboards had not yet been deployed because the cluster-logging-operator did not correctly compare existing and desired configmaps containing the dashboard. This update fixes the logic by adding a the dashboard unique hash value to the object labels.(link:https://issues.redhat.com/browse/LOG-2066[LOG-2066])
```